app/views/availabilities/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <b>Person:</b>
+  <%= link_to @availability.person.fullname, @availability.person %>
+</p>
+
+<% if @availability.event.present? %>
+  <p><b>Event:</b><%= link_to @availability.event.title, @availability.event %></p>
+<% end %>
+
+<p>
+  <strong>Start time:</strong>
+  <%= @availability.start_time %>
+</p>
+
+<p>
+  <strong>End time:</strong>
+  <%= @availability.end_time %>
```

Comment:
Is there a way to get the javascript to automatically bring the end time to no earlier than the start time?